nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-core/src/main/java/org/apache/nifi/controller/scheduling/StandardProcessScheduler.java

```diff
@@ -288,171 +284,74 @@ public void run() {
     }
 
     /**
-     * Starts scheduling the given processor to run after invoking all methods on the underlying {@link org.apache.nifi.processor.Processor
-     * FlowFileProcessor} that are annotated with the {@link OnScheduled} annotation.
+     * Starts the given {@link Processor} by invoking its
+     * {@link ProcessorNode#start(ScheduledExecutorService, long, org.apache.nifi.processor.ProcessContext, Runnable)}
+     * .
+     * @see StandardProcessorNode#start(ScheduledExecutorService, long,
+     *      org.apache.nifi.processor.ProcessContext, Runnable).
      */
     @Override
     public synchronized void startProcessor(final ProcessorNode procNode) {
-        if (procNode.getScheduledState() == ScheduledState.DISABLED) {
```

Comment:
A bit confused as to where this went.  Stepped through the associated implementations but not seeing any place where this is validated.